### PR TITLE
Add sass

### PR DESCRIPTION
**Pull request by gussiciliano**
### Summary
- Add gulp
- Add gulp sass watcher
- example-site and example-site.min there are examples auto generates by dotnet command

### ISSUE
https://github.com/Wolox/netcore-bootstrap/issues/2

### CODE REVIEW

**Comment by gussiciliano** [comment]
@SKOLZ In this PR there are the config for use Scss in a Net Core project, Can you check the gulp.js file?

File: package.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "devDependencies": {
+        "gulp": "^3.9.1",
+        "gulp-concat": "2.6.1",
+        "gulp-cssmin": "0.1.7",
+        "gulp-uglify": "2.0.1",
+        "rimraf": "2.6.1"
+    },
+    "dependencies": {
+        "gulp-sass": "^3.1.0"
```
Comment (mishuagopian) [inline package.json]: This should be dev too, right?

Comment (gussiciliano) [reply package.json]: Yes :)

File: README.md [hunk 2]
```diff
@@ -36,10 +36,25 @@ Run the following command from rootpath of the project to get the specified pack
...
+### Using SCSS
+For use Scss files you need to run the following comands:
+```bash
+    gulp
+```
+
+If you don't have gulp run these comands:
+```bash
+    npm install gulp
+    npm install gulp-sass
```
[the remark this answers is not in the document]
Comment (gussiciliano) [reply README.md]: should be npm install gulp-sass --save-dev?

**Pull request merged**